File: dayFour/java/afterBreak/TaskThree.java
import java.util.Scanner;

public class TaskThree{

    public static void main(String[] args){
    
        Scanner input = new Scanner(System.in);
        int total = 0;
        
        for(int count = 1; count <= 10; count++){
            System.out.println("Enter a score: ");
            int number = input.nextInt();
            total += number;
        }
        int average = total / 10;
        System.out.println("The sum of the ten scores is "+total+ " and the average is "+average);
    }
}

File: dayFour/javaScript/taskThree.js
let total = 0;

for(let count = 1; count <= 10; count++){
    
    const scanner = require("prompt-sync")();
    let number = Number(scanner ("Enter score: "));
    total = total + number;
}
let average = total / 10;

console.log("The sum of the ten scores is "+total+" and the average is "average);
